pkg/dataflatten/jwt.go

```diff
@@ -0,0 +1,49 @@
+package dataflatten
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/golang-jwt/jwt/v5"
+)
+
+// JWTFile adds support for the kolide_jwt table, which allows parsing
+// a file containing a JWT. Note that the kolide_jwt table does not handle
+// verification - this is a utility table for convenience.
+func JWTFile(file string, opts ...FlattenOpts) ([]Row, error) {
+	return flattenJWT(file, opts...)
+}
+
+func flattenJWT(path string, opts ...FlattenOpts) ([]Row, error) {
+	results := map[string]interface{}{}
+	jwtFH, err := os.Open(path)
+	if err != nil {
+		return nil, fmt.Errorf("unable to access file: %w", err)
+	}
+
+	defer jwtFH.Close()
+
+	tokenRaw, err := io.ReadAll(jwtFH)
+	if err != nil {
+		return nil, fmt.Errorf("unable to read JWT: %w", err)
+	}
+
+	// attempt decode into the generic (default) MapClaims struct to ensure we capture
+	// any claims data that might be useful
+	token, _, err := new(jwt.Parser).ParseUnverified(string(tokenRaw), jwt.MapClaims{})
```

Comment:
What's the second return value?

Reply:
this is `parts = strings.Split(tokenString, ".")` - we are essentially just base64 decoding `parts[1]` in the remaining bits

Reply:
Is there anything to extract from the other parts?

Reply:
we will get the header now too per the conversation below. I can include the raw signature as well if you think that would be helpful?

Reply:
I don't think we should return the raw signature. Or the contents of the file